server/client.py

```diff
@@ -96,7 +99,20 @@ def _gen_secret_key_map(self, org_id):
 
     @_handle_api_errors
     def _get_secret_from_client(self, secret_uuid: str):
-            return self.bws_client.secrets().get(secret_uuid).data
+        data = self.bws_client.secrets().get(secret_uuid).data
+        try:
+            data.value = json.loads(data.value)
+            return data
+        except json.JSONDecodeError:
+            logging.info("json parse failed")
+        try:
+            data.value = yaml.safe_load(data.value)
+            return data
+        except yaml.YAMLError:
+            logging.info("yaml parse failed")
```

Comment:
Thoughts on specifying the secret for which parsing failed?
```suggestion
            logging.info(f"json parse failed for secret {secret_uuid}")
        try:
            data.value = yaml.safe_load(data.value)
            return data
        except yaml.YAMLError:
            logging.info(f"yaml parse failed for secret {secret_uuid}")
```

Reply:
hmmmm, i like the idea but i belive its contextualised in you see the url hit and that has the secret info, so is this redundant? i have no issue doing this if you dont agree :3